src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    @Override
+    public String toString() {
+        return "color: " + this.color + ", number: " + this.number;
```

Comment:
there are not requirement for custom toString, so please use default one (right click -> gererate -> toString)